tests/platform_tests/fwutil/test_fwutil.py

```diff
@@ -0,0 +1,105 @@
+import re
+import pytest
+import os
+import json
+
+from fwutil_common import call_fwutil, show_firmware, upload_platform, find_pattern
+
+DEVICES_PATH="/usr/share/sonic/device"
+
+def test_fwutil_show(duthost):
+    """Basic Validation that fwutil show runs without error"""
+    assert show_firmware(duthost)
```

Comment:
@alexrallen what if some of the platform components is missing in this output? How we are going to verify it? Maybe we can leverage default `platform_components.json`?

Reply:
Good thought, implemented this.